### ISSUE TITLE
Android deep linking - app-to-app communication without the web experience

### ISSUE THREAD

**Issue by valeriik** [osmandapp/OsmAnd-iOS#1487]
I cannot find other deep link then this:
`https://osmand.net/go.html?lat=${latitude}&lon=${longitude}&z=17`
And when I try to open OsmAnd via this deep link on Android OS offers me the choice to open the app through the installed application OR through a web browser. But HOW to open OsmAnd app directly and without this choice?

For example Waze app has next deep link:
waze://ul?ll=${latitude},${longitude}&navigate=yes
and therefore Android OS opens Waze app without any additional questions from the user what he/she wants to open this link.

Explanation is here:
https://developers.google.com/waze/deeplinks/
"...If you're sure that the Waze app is installed, you can use the URL "waze://" for app-to-app communication without the web experience. ..."

**Comment by vshcherb** [comment]
I think it's now solved differently cause we have site linked directly to app, so it might that issue is overrated

**Comment by Zirochkabila** [comment]
If you click on the link https://osmand.net/go?lat=50.43829&lon=30.50322&z=20 - OsmAnd will automatically open

**Comment by valeriik** [comment]
> If you click on the link https://osmand.net/go?lat=50.43829&lon=30.50322&z=20 - OsmAnd will automatically open

No, for example on my Android 12 it is not true. When I clicked FIRST TIME on this link I see this popup
![telegram-cloud-photo-size-2-5375111675449036017-y](https://user-images.githubusercontent.com/24571089/197241136-023d960e-d1d6-4454-8f72-60658755108c.jpg)

**Comment by Zirochkabila** [comment]
@valeriik Try to make it this way: Press on map > Share > Send > Select a messenger > Open link in messenger

**Comment by Zirochkabila** [comment]
https://github.com/osmandapp/OsmAnd-iOS/issues/2205#issuecomment-1293617801
